### PR TITLE
Line image extraction from MAR and getting line image and transcription mapping

### PR DESCRIPTION
**Pull request by aarora8**

### CODE REVIEW

**Comment by danpovey** [comment]
Can you explain what this changes?  I though we had already switched over madcat arabic to be based on the non-axis-oriented MARs for each line, rather than the axis-oriented bounding boxes.

**Comment by aarora8** [comment]
Sorry, it is regarding creating line images from the segmentation output. And creating a mapping between the line images and transcription. The text corresponding to the reference box that had the largest IoU overlap with that box is mapped with the reference box.

File: egs/madcat_arabic/v1/run.sh [hunk 1]
```diff
@@ -71,11 +72,23 @@ fi
...
     echo "getting score based on comparing mask image... Date: $(date)."
-    scoring/score.py data/test/mask $dir/segment/mask $dir/segment/result.txt
+    scoring/score.py data/test/mask $dir/segment/mask $dir/segment \
+      --mar-text-mapping data/test/mar_transcription_mapping.txt
   fi
 fi
+
+if [ $stage -le 5 ]; then
+  echo "extracting line images from page image using mar information. Date: $(date)."
+  mkdir -p $dir/segment/lines
```
Comment (danpovey) [inline egs/madcat_arabic/v1/run.sh]: where does this file madcat.test.raw.lineid come from?
Let's not say "mar information", let's say "waldo segmentation output" if that's where it comes from,
as "mar information" is very ambiguous.

Comment (aarora8) [reply egs/madcat_arabic/v1/run.sh]: ok, thanks. I will change it. madcat.test.raw.lineid, is a test split, we are getting it from openslr (http://www.openslr.org/resources/48/madcat.test.raw.lineid)

**Comment by danpovey** [comment]
This is great progress.  If you think it's ready I can merge it now.

**Comment by aarora8** [comment]
Thank you, it is ok to merge.

**Pull request merged**